src/System.Net.Http/src/project.json

```diff
@@ -11,7 +11,8 @@
     "System.Runtime.Extensions": "4.0.10",
     "System.Runtime.Handles": "4.0.0",
     "System.Runtime.InteropServices": "4.0.20",
-    "System.Security.Cryptography.X509Certificates": "4.0.0-*",
+    "System.Security.Cryptography.X509Certificates": "4.0.0-beta-23509",
```

Comment:
Why are you referencing a specific beta version?

It probably doesn't matter, we should be doing a "-beta-" to "-rc2-" scrub soon...